internal/expr/parser.go

```diff
@@ -486,24 +521,76 @@ func (p *Parser) parseIdentifier() (string, bool) {
 	return "", false
 }
 
-// parseColumn parses a column made up of name bytes, optionally dot-prefixed by
+// skipEnclosedParentheses starts from a opening parenthesis '(' and skips until
+// its closing ')', taking into account comments and string literals in between.
```

Comment:
Maybe point out that it also takes into account nested parenthesis inside the main pair.